__tests__/index.test.js

```diff
@@ -42,3 +42,39 @@ describe("/api", () => {
       });
   });
 });
+describe("/api/articles/:article_id", () => {
+  test("200: return article object by it's id, with 8 correct properties", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((data) => {
+        const articles = data.body.article;
+        expect(articles.length).toBe(1);
+        expect(articles[0]).toHaveProperty("title");
+        expect(articles[0]).toHaveProperty("article_id");
+        expect(articles[0]).toHaveProperty("body");
+        expect(articles[0]).toHaveProperty("topic");
+        expect(articles[0]).toHaveProperty("created_at");
+        expect(articles[0]).toHaveProperty("votes");
+        expect(articles[0]).toHaveProperty("article_img_url");
+      });
```

Comment:
You should know what this article looks like, at the very least you can check that the article_id is 1!